experiment_api/experiment.py

```diff
@@ -224,35 +231,11 @@ def __eq__(self, other):
         return self.name == other.name
 
 # -----------------------------------------------------------------------------
-#  Helper functions
-# -----------------------------------------------------------------------------
-
-
-def get_numeric(string):
-    """ Extract the numeric value in a string.
-    Args:
-        string
-    Returns:
-        a string with only the numeric value extracted
-    """
-    return re.sub('[^0-9]', '', string)
-
-
-def should_drop(drop_percentage):
-    """ Based on the given percentage, provide an answer
-    whether or not to drop the image.
-    Args:
-        drop_percentage: the likelihood of a drop in the form of a float from [0,1]
-    Returns:
-        a boolean whether to drop or not drop the image
-    """
-    return random.random() < drop_percentage
-
-# -----------------------------------------------------------------------------
 #  Execution example
 # -----------------------------------------------------------------------------
 
 
+@timewrapper
```

Comment:
I'll just comment it for now in case we wanted to profile it again in the future.